README.md

```diff
@@ -43,22 +44,19 @@ USER_EDIPI=0000000001
 
 - Navigate to the project root and run the following.
 ```
-npm install
+yarn install
 
 # Optional if using docker-compose
-npm run start-db  
+yarn run start-db  
 
-npm run seed-dev
-npm run dev
+yarn run seed-dev
+yarn run dev
 ```
 This will start both the frontend and the backend.
 
-In the logs you’ll see [0] and [1] at the start of each line.
-[0] is from the server and [1] is from the client.
-
 If you need to stop or reset the docker-compose dev database:
```

Comment:
In the new setup these now say `[server]` and `[client]`, and have pretty colors too!